1_fetch.R

```diff
@@ -46,9 +47,39 @@ p1_targets_list <- list(
   tar_target(
     p1_nhd_reaches,
     download_nhdplus_flowlines(p1_drb_comids_all_tribs$COMID)
-  )
-
+  ),
+  
+  # STATSGO SOIL Characteristics
+  ## get selected child items nhdv2 STATSGO Soil Characteristics
+  ## 1) Text attributes and 2) Layer attributes
+  tar_target(
+    p1_selected_statsgo_sbid_children,
+    item_list_children(sb_id = nhd_statsgo_parent_sbid) %>% 
+      Filter(function(x){str_detect(x[['title']],'Text|Layer')},
+             .)
+    ),
   
+  ## download selected CONUS STATSGO datasets from Science base
+  tar_target(
+    p1_download_statsgo_text_layer_attr,
```

Comment:
```suggestion
    p1_download_statsgo_text_layer_zip,
```

Minor suggestion for the target name to make it apparent what the expected output looks like. If you do opt to go with this change, you'll need to also update the downstream targets that reference `p1_download_statsgo_text_layer_attr`.

Reply:
changed.